website/docs/about/overview.md

```diff
@@ -1,9 +1,52 @@
 ---
 title: Overview
-description: This is the (missing) long description of the component, that will come from the frontmatter attributes
+description: Learn about the HashiCorp Design System, why it exists, and the strategy behind the system.
 order: 101
 ---
 
-## Lorem ipsum dolor
+## What is a design system?
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Diam diam mi etiam mollis tortor vestibulum. Felis, arcu amet, nibh eget eget etiam orci aliquet. Varius facilisis magna faucibus commodo iaculis faucibus aliquet.
\ No newline at end of file
+A design system is a set of guidelines, standards, assets, and processes to help organizations rapidly design, build, and deploy products in a consistent manner.
+
+Design systems are often equated with a UI kit or component library; in reality an effective design system spans the entire lifecycle of a product through ideation to iteration and everything in between.
+
+### Architecture
+
+While the scale of a design system largely depends on the needs of those consuming it, the fundamental structure and architecture remains largely the same across systems.
+
+- **Visual Language**: colors, typography, icons, imagery, content strategy
+- **Components**: component design, component development, tokens, development standards
+- **Documentation**: education, issue tracking, contribution, content management, processes
+
+At HaschiCorp we structure our components, icons, and tokens in a [monorepo](https://github.com/hashicorp/design-system) using yarn workspaces and publish a collection of libraries in [Figma](https://www.figma.com/files/team/1030156573400567478) for designers.
```

Comment:
```suggestion
At HaschiCorp we structure our components, icons, and design tokens in a [monorepo](https://github.com/hashicorp/design-system) using yarn workspaces and publish a collection of libraries in [Figma](https://www.figma.com/files/team/1030156573400567478) for designers.
```

Suggestion: say "design tokens" instead of just "tokens" since "tokens" is an overloaded word.